R/fetch_pdb.R

```diff
@@ -0,0 +1,86 @@
+#' Fetch protein structure data from RCSB
+#'
+#' Fetches protein structural data from the RCSB database.
+#'
+#' @param pdb_ids A character vector of pdb identifiers
+#' @param columns A character vector of data columns that should be imported from RCSB (all possible columns can be found here: https://www.rcsb.org/pdb/results/reportField.do)
+#' @param batchsize Size of batch of proteins for a single query
+#' @param show_progress Logical, if true, a progress bar will be shown
+#'
+#' @return A data frame that contains all structural data specified in \code{columns} for the pdb structures provided.
+#' @import janitor
+#' @import progress
+#' @import purrr
+#' @import RCurl
+#' @import dplyr
+#' @importFrom data.table fread
+#' @importFrom magrittr %>%
+#' @importFrom rlang .data
+#' @export
+#'
+#' @examples
+#' \dontrun{
+#' fetch_uniprot(c("3OFN", "3ZIA", "3ZRY"))
```

Comment:
wrong example (fetch_pdb instead of fetch_uniprot)

Reply:
Corrected it!